.github/workflows/ci.yml

```diff
@@ -27,6 +27,11 @@ jobs:
                         php: '8.2'
```

Comment:
@dannyvw can you remove this 8.2 block? 8.2 will be tested in the general php matrix and we can just test the latest php (8.3) with the lts release!

Reply:
@acrobat Done